### DISCUSSION TITLE
The function of shouldGenerateTitle according to the request does not seem to be in effect

### DISCUSSION THREAD

**Discussion by Henry910** [waylaidwanderer/node-chatgpt-api#311] [Help]
I am using version v1.34, the function of shouldGenerateTitle according to the request does not seem to be in effect.

The code is as follows:
"settings.apiOptions.generateTitles: true;"  // or settings.apiOptions.generateTitles: false | and the result is the same

The requested BODY is written as follows:
{
  "message": "Write a poem",
  "conversationId": "",
  "parentMessageId": "",
  "clientOptions": {
      "modelOptions": {
      "model": "gpt-3.5-turbo",
      "max_tokens": 200,
      "temperature": 0.8
      }
  },
  "maxContextTokens": 1024,
  "maxPromptTokens": 200,
  "userLabel": "User",
  "chatGptLabel": "ChatGPT",
  "promptPrefix": "You are willing to help others",
  "shouldGenerateTitle": true  // or delete this row, and the result is the same
}

But there is no parameter for 'title' in the reply result, excuse me, is there an error?

**Comment by waylaidwanderer** [comment]
It seems to be working for me, are you _sure_ you're on 1.34?

**Reply by Henry910** [reply]
Yes, i am sure, v1.34.  
In the BODY request parameter with 'promptPrefix', it also does not work. :)
